firmware/library/L1_Drivers/uart.hpp

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+class UARTInterface
+{
+ public:
+    virtual void setBaud(unsigned int baudrate) = 0;
```

Comment:
Follow Google C++ style guide. Should be `SetBaud`.
`int` has different size depending on what system you are compiling for, please use <cstdint>